packages/iov-lisk/src/liskclient.spec.ts

```diff
@@ -0,0 +1,122 @@
+    const client = new LiskClient(base);
+    const query: BcpAccountQuery = { address: Encoding.toAscii("15683599531721344316L") as Address };
+    const account = await client.getAccount(query);
+    expect(account.data[0].address).toEqual(Encoding.toAscii("15683599531721344316L"));
+    expect(account.data[0].balance[0].tokenTicker).toEqual("LSK");
+    expect(account.data[0].balance[0].sigFigs).toEqual(8);
+    expect(account.data[0].balance[0].whole).toEqual(15);
+    expect(account.data[0].balance[0].fractional).toEqual(48687542);
+  });
+
+  it("can get nonce", async () => {
+    const client = new LiskClient(base);
+    const query: BcpAccountQuery = { address: Encoding.toAscii("15683599531721344316L") as Address };
+    const nonce = await client.getNonce(query);
+
+    expect(nonce.data[0].address).toEqual(Encoding.toAscii("15683599531721344316L"));
+    // nonce is current timestamp +/- one second
+    expect(nonce.data[0].nonce.toNumber()).toBeGreaterThanOrEqual(Date.now() / 1000 - 1);
+    expect(nonce.data[0].nonce.toNumber()).toBeLessThanOrEqual(Date.now() / 1000 + 1);
+  });
+
+  it(
+    "can post transaction",
+    async () => {
+      pendingWithoutLongRunning();
+
+      const liskTestnet = "da3ed6a45429278bac2666961289ca17ad86595d33b31037615d4b8e8f158bba" as ChainId;
+
+      const entry = new LiskKeyringEntry();
+      const mainIdentity = await entry.createIdentity(
+        "oxygen fall sure lava energy veteran enroll frown question detail include maximum",
+      );
+
+      const recipientAddress = Encoding.toAscii("6076671634347365051L") as Address;
+
+      const sendTx: SendTx = {
+        kind: TransactionKind.Send,
+        chainId: liskTestnet,
+        signer: mainIdentity.pubkey,
+        recipient: recipientAddress,
+        memo: "We ❤️ developers – iov.one",
+        amount: {
+          whole: 1,
+          fractional: 44550000,
+          tokenTicker: "LSK" as TokenTicker,
+        },
+      };
+
+      // Encode creation timestamp into nonce
+      const nonce = generateNonce();
+      const signingJob = liskCodec.bytesToSign(sendTx, nonce);
+      const signature = await entry.createTransactionSignature(
+        mainIdentity,
+        signingJob.bytes,
+        signingJob.prehashType,
+        liskTestnet,
+      );
+
+      const signedTransaction = {
+        transaction: sendTx,
+        primarySignature: {
+          nonce: nonce,
+          publicKey: mainIdentity.pubkey,
+          signature: signature,
+        },
+        otherSignatures: [],
+      };
+      const bytesToPost = liskCodec.bytesToPost(signedTransaction);
+
+      const client = new LiskClient(base);
+      const result = await client.postTx(bytesToPost);
+      expect(result).toBeTruthy();
+      expect(result.metadata.height).toBeDefined();
+      expect(result.metadata.height).toBeGreaterThan(6000000);
+      expect(result.metadata.height).toBeLessThan(8000000);
+    },
+    40 * 1000,
```

Comment:
oof, this can take 40 seconds?

Reply:
If I want to have feedback like height, I need to sleep 20 seconds between submission and status query. So it should not take longer than ~22 seconds